src/components/FormBook.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const FormBook = () => (
+  <div className="form-container">
+    <h4>
+      ADD NEW BOOK
+    </h4>
+    <form className="row gy-2 gx-3 align-items-cente">
+      <div className="col-auto">
+        <div className="input-group">
+          <input type="text" className="form-control" id="title" placeholder="Book title" />
+        </div>
+      </div>
+      <div className="col-auto">
+        <select className="form-control" id="sel1" placeholder="category">
+          <option>Action</option>
+          <option>Romance</option>
+          <option>Fantasy</option>
+          <option>Mistery</option>
+        </select>
+      </div>
+
+      <div className="col-auto">
+        <button type="submit" className="btn btn-primary">ADD BOOK</button>
+      </div>
+    </form>
```

Comment:
- [ ] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.